src/test/mock/SimpleStateBridge.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {IBridge} from "../../interfaces/IBridge.sol";
+
+/// @title Simple mock of the state bridge to make tests compile 
```

Comment:
```suggestion
/// @title State Bridge Mock
/// @notice This purely exists to allow tests to compile and does not have any functionality.
```

Please make sure this wraps at the appropriate 100 chars as I don't have a ruler on GitHub.